### ISSUE TITLE
twisted/web/failure.xhtml isn't in installed Twisteds on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8047]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8047|
|Type|enhancement|
|Created|2015-10-13 01:38:12Z|
|Branch|https://github.com/twisted/twisted/tree/failurexhtml-8047-2|

<details><summary>Searchable metadata</summary>

```
trac-id__8047 8047
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_failurexhtml_8047_2 branches/failurexhtml-8047-2
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1444700292034283 1444700292034283
changetime__1444711846123706 1444711846123706
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [45764]) Branching to failurexhtml-8047.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [45770]) Branching to failurexhtml-8047-2.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Builders spun, please review.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @hawkowl|
|-|-|

Only one feedback point:

  1. One pyflakes error: https://buildbot.twistedmatrix.com/builders/pyflakes/builds/3083/steps/pyflakes/logs/new%20pyflakes%20errors

and it's a shame that we couldn't see a failing build for this before we saw a passing one.  But I suppose the whole point of doing this is to have a quick workaround for now and address the build issues later.

Can you file a ticket to address the `data_files` / `package_data` issue at some later date, fix the pyflakes error, and merge?

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45773]) Merge failurexhtml-8047-2: Inline twisted.web.util's failure template

Author: hawkowl
Reviewer: glyph
Fixes: [#8047](https://github.com/twisted/twisted/issues/8047)
